modules/http/http-grammar.ym

```diff
@@ -73,10 +76,17 @@
 %token KW_WORKERS
 %token KW_ACCEPT_REDIRECTS
 %token KW_AUTH_HEADER
+%token KW_ON_HTTP_RESPONSE
```

Comment:
why don't we call this response-action() in the configuration file as well?

e.g. instead of `on-http-response()` we would just use `response-action(404 => retry)` I think that's easier to understand. I find the "on-" variant a bit "programmery".

Reply:
You are right.This came from the "era" when I intended to add many filter parameters besides the http status code, likeheader/body matching. As we concluded earlier, we would support these within the actions. So the `on-` part does not make sense anymore.

Reply:
Fixed.